core/src/main/java/io/undertow/UndertowLogger.java

```diff
@@ -393,7 +393,7 @@ void nodeConfigCreated(URI connectionURI, String balancer, String domain, String
     @Message(id = 5084, value = "Attempted to write %s bytes however content-length has been set to %s")
     IOException dataLargerThanContentLength(long totalToWrite, long responseContentLength);
 
-    @LogMessage(level = ERROR)
+    @LogMessage(level = WARN)
```

Comment:
Thoughts on moving this down to INFO level given we know it occurs when a rst_stream frame is received?

Reply:
No longer necessary, I root caused and fixed the bug.